Makefile

```diff
@@ -48,6 +48,14 @@ ifeq ($(STATIC),1)
 GOFLAGS  += -a -tags netgo -ldflags '-extldflags "-lm -lstdc++ -static"'
 endif
 
+LDFLAGS := 
+ifeq ($(RELEASE),1)
+LDFLAGS := -X github.com/cockroachdb/cockroach/util.buildSHA "$(shell git rev-parse HEAD)"
```

Comment:
Can we do this inside the `build` target so `make` doesn't have to run multiple git processes at startup whose output may never be used?